spec/truffle.mspec

```diff
@@ -18,7 +22,8 @@ class MSpecScript
 
   set :target, "#{TRUFFLERUBY_DIR}/bin/truffleruby"
 
-  unless ARGV.include?('-t')  # No flags set if Ruby binary specified via -t.
+  # No flags set if a Ruby binary is specified via -t
+  if !child_process? and !ARGV.include?('-t')
```

Comment:
I don't know if we have a Rubocop rule for this, but I thought we settled on `&&` for simple boolean expressions like this.